### ISSUE TITLE
The Elden Ring Feature Request

### ISSUE THREAD

**Issue by YUSTIAN0423** [DiUS/java-faker#707]
**Is your feature request related to a problem? Please describe.**
The Elden Ring is a hot game released recently . It is excellent , maybe you could add features of this game. 

**Describe the solution you'd like**
Maybe some features including profession, magic, weapon .etc

**Comment by Lance-Azrael** [comment]
Hello @YUSTIAN0423 , we also like this game! Our group are interested in this issue, and we will try to make it.
---- SE_SUSTech, group: Lanrand

**Comment by YUSTIAN0423** [comment]
@Lance-Azrael Perfect! Thanks to your reply, I'm looking forward to your works.

**Comment by bodiam** [comment]
@YUSTIAN0423 it's a bot :-(

If you want this, please make a PR to Datafaker.net, which is a fork of this library, but maintained. It's quite easy to create a provider like this, happy to help

**Comment by Lance-Azrael** [comment]
I am not robot :(
We are students from SUSTech, SE class. We have a mission which need us to fix more than 10 issues from projects in GitHub, and our group choose data faker project. We will finish these mission before July. Thank you for your understanding!

> @YUSTIAN0423 it's a bot :-(
> 
> If you want this, please make a PR to Datafaker.net, which is a fork of this library, but maintained. It's quite easy to create a provider like this, happy to help

**Comment by Lance-Azrael** [comment]
java faker... wrong typo

**Comment by YUSTIAN0423** [comment]
@bodiam Thank you for your help and advice!
But don't worry, because I've already made contact with @Lance-Azrael . They are a course team of five students. I have seen contributions made by other course teams on other issues of this faker project and I think it is worth a try to cooperate with them.
I will try to complete this issue together with them. They also mentioned datafaker to me, but I didn't know much about it. I'm going to learn it.
Thank you again for your attention and help.

**Comment by bodiam** [comment]
@Lance-Azrael oh, I'm sorry. This project however is quite dead. It has lots of issues, most of it are fixed in Datafaker.net, which is a maintained fork of this library.

If you want to contribute to Datafaker.net instead, we'd welcome all contributions, and it would be great to also release them to the world, and I'm happy to provide coaching if needed (I'm a developer myself with 20+ years of experience).

We release datafaker every month, while this project hasn't had a release in over 2 years.

**Comment by bodiam** [comment]
@YUSTIAN0423  ah, that's alright! Datafaker is a fork of this project, with the same source files, same api, etc, but it has all the security issues fixed, is much faster, has more providers, has support for custom validations, has support for csv, yml, json, etc. 

Plus, it's very actively maintained!

**Comment by Lance-Azrael** [comment]
@bodiam Oh I understand. Thank you very much! 
Can we modify the Datafaker project according to the requirements of some issues in this Javafaker project? Because I see a lot of problems in the Javafaker feature issue, especially with the Fakers requirement. If possible, we will explain the situation under the issue we promised to help.

**Comment by bodiam** [comment]
@Lance-Azrael absolutely! I think several issues have been fixed already, but if you find any issues, or want to contribute new providers or new features, that would be very welcome.

**Comment by bodiam** [comment]
@Lance-Azrael I've also added 2 issues to datafaker, which can be implemented as a great first issue. Again, happy to help with any help you may need.

**Comment by Lance-Azrael** [comment]
@bodiam I see! Thank you a lot! (●'◡'●)
